### ISSUE TITLE
Remove predetermined path in cfg.hpp

### ISSUE THREAD

**Issue by kehrazy** [es3n1n/portal2-internal#20]
rn the _dir is `c:\\portal2-internal` which, well, fucking sucks for a lot of reasons

1. can we force the configs to a user folder, so that in unix-like systems down the line the config doesn't just create a folder in root directory?

2. can std::filesystem resolve paths in compiletime? if so, resolve in compile time we should

**Comment by es3n1n** [comment]
good point, actually i think we should just create a directory inside of the game dir and store configs there
as for unix-like systems i don't even think that we can compile it on them and use them since a hella lot of stuff is using winapi here, porting it to linux\mac sounds like a cool idea tho, i'll take it
